pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "How the W3C Web of Things brings JSON Schema to the Internet of Things"
+date: 2022-08-17T06:00:00+00:00
+type: Case Study
+authors:
+  - name: Juan Cruz Viotti
+    photo: /img/avatars/jviotti.webp
+    link: https://twitter.com/jviottidc
+excerpt: "Using JSON Schema at the W3C Web of Things to counter the fragmentation of the Internet of Things"
+language: en
+---
+
+## Challenge
+
+The Internet of Things (IoT) is the network of physical "smart" objects that
+exchange data with other devices over the Internet. While communication and
+interoperability are by definition the crux of the Internet of Things, the
+emergence of custom or proprietary solutions results in devices that cannot
+talk to each other due to differences in data interchange mechanisms.
+
+To integrate these disparate devices, developers must work with a growing set
+of protocols, serialization formats and API specifications. This results in
+repetitive, non-scalable and error-prone work that is difficult to automate.
+
+While technologies like OpenAPI and AsyncAPI largely solve this problem in the
+context of Web APIs, they fall short for describing networks of non-HTTP
+multi-protocol devices and do not consider different modes of interaction based
+on their meaning in the physical world.
+
+## Solution
+
+To solve these problems, the W3C Web of Things works on providing standardized
+building blocks that make use of JSON Schema.
+
+JSON Schema is used for validating abstractions of physical entities called
+[Thing Descriptions][thing-description], and to model and describe data sent by
+Internet of Things consumers and producers in a multi-protocol manner.
+
+We are on-board with JSON Schema since Draft 4, and even the first draft
```

Comment:
Shouldn't we use another subject than "we"? There are other places with it as well.

Reply:
True. Other case studies use "We" only in quotations. I'll fix it everywhere

Reply:
Done